### PR TITLE
add master detail relationship support

### LINKED ISSUE #4146: Add support for Aggregation in ER Diagrams
### Proposal

There is a special case of a parent-child relationship (one to many), where the parent side is represented using a class aggregation symbol and the many side is a crow feet(supported). Can the aggregation symbol supported in class diagram be extended to ER diagram. This will be very useful to model more advanced relationships.

### Example

_No response_

### Screenshots

m-d

### PR DESCRIPTION
**Pull request by tcbuzor**
## :bookmark_tabs: Summary

Added support for a special relationship type known as Master-Detail or Parent-Child relationship. A master detail or parent child relationship is a strongly-coupled relationship type between two objects, where the parent controls the record ownership of child records.

Resolves #4146 

## :straight_ruler: Design Decisions

Describe the way your implementation works or what design decisions you made if applicable.
Example Relationship Syntax :
A parent child relationship between Project and Team Member can be expressed as follows:
PROJECT u--|{ TeamMember : child 

### :clipboard: Tasks

Make sure you

- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [x] :computer: have added unit/e2e tests (if appropriate)
- [ ] :notebook: have added documentation (if appropriate)
- [x] :bookmark: targeted `develop` branch

### CODE REVIEW

**Comment by tcbuzor** [comment]
This is showing as composition right now. I'd like to keep it as true aggregation(no shaded)
Screenshot 2023-04-30 at 2 05 07 PM

**Comment by sidharthv96** [comment]
Classdiagram seems to use this CSS to specify fill and stroke. Can you try using that?

https://github.com/mermaid-js/mermaid/blob/33c81d557ee41dda9aa2326b103bfd69b1bdcff9/packages/mermaid/src/diagrams/class/styles.js#L123-L127

**Comment by tcbuzor** [comment]
Fixed with the following added to style..js
 #MD_PARENT_START {
    fill: #f5f5f5 !important;
    stroke: ${options.lineColor} !important;
    stroke-width: 1;
  }
  #MD_PARENT_END {
    fill: #f5f5f5 !important;
    stroke: ${options.lineColor} !important;
    stroke-width: 1;
  }

**Comment by tcbuzor** [comment]
@sidharthv96 , can you review this PR and merge to master?

**Comment by knsv** [comment]
This looks great!, Thanks @tcbuzor !

File: packages/mermaid/src/diagrams/er/styles.js [hunk 1]
```diff
@@ -33,6 +33,17 @@ const getStyles = (options) =>
     font-size: 18px;
     fill: ${options.textColor};
   }    
+  #MD_PARENT_START {
+    fill: #f5f5f5 !important;
+    stroke: ${options.lineColor} !important;
+    stroke-width: 1;
+  }
+  #MD_PARENT_END {
```
Comment (sidharthv96) [inline packages/mermaid/src/diagrams/er/styles.js:41]: This is not consistent with the other style selector naming conventions. Please use `.lowerCamelCase` class names.

**Pull request merged**